hs/src/Reach/AST/DLBase.hs

```diff
@@ -639,6 +638,7 @@ data DLExpr
   | DLE_GetUntrackedFunds SrcLoc (Maybe DLArg) DLArg
   | DLE_FromSome SrcLoc DLArg DLArg
   -- Maybe try to generalize FromSome into a Match
+  | DLE_BalanceInit Int DLVar
```

Comment:
This is for the initial creation of the balance array. At the start of the program, we don't know how big the array will be yet, so this will transform to an actual array after evaluation. DLVar is `<None null>` which will be used to initialize every spot in the array.

Reply:
Why are they maybes, rather than just UInts? And what's the number?